lib/menu.rb

```diff
@@ -0,0 +1,24 @@
+require_relative 'menu_item'
+
+class Menu
+
+  attr_reader :list
+
+  def initialize
+    @list = Hash.new
+  end
+
+  def add_new_item(name, price)
+    @list[name] = MenuItem.new(name, price)
```

Comment:
It might be worth thinking about what happens if you try and add an item for a name that already exists.